packages/rustcat/PKGBUILD

```diff
@@ -17,7 +16,7 @@ source=("git+https://github.com/robiot/$pkgname.git")
 pkgver() {
   cd $pkgname
 
-  echo $(git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g')
+  git describe --long --tags | sed 's/^v//;s/\([^-]*-g\)/r\1/;s/-/./g'
```

Comment:
this is not how it works, see our PKGBUILD templates https://github.com/BlackArch/blackarch-pkgbuilds/blob/a8f597f28e9ec58ac8aa300f5825237c2aaf46b8/PKGBUILD-generic-git#L17-L25